deblur/parallel_deblur.py

```diff
@@ -0,0 +1,71 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015, The Deblur Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+from __future__ import division
+
+from os.path import basename, join, split, splitext
+from os import mkdir
+
+from bfillings.sortmerna_v2 import build_database_sortmerna
+from skbio.parse.sequences import parse_fasta
+
+from qiime.parallel.util import ParallelWrapper
+
+
+class ParallelDeblur(ParallelWrapper):
```

Comment:
Launching on clusters requires knowledge of the submission mechanisms which is the primary abstraction that `ParallelWrapper` provides. But, that's only important if the intent is to run a large number of jobs in parallel